src/pool/srv_pool.c

```diff
@@ -368,6 +368,15 @@ pool_prop_write(struct rdb_tx *tx, const rdb_path_t *kvs, daos_prop_t *prop)
 		entry = &prop->dpp_entries[i];
 		switch (entry->dpe_type) {
 		case DAOS_PROP_PO_LABEL:
+			if (entry->dpe_str == NULL ||
+			    strlen(entry->dpe_str) == 0) {
+				entry = daos_prop_entry_get(&pool_prop_default,
+							    entry->dpe_type);
+				if (entry == NULL) {
```

Comment:
is D_ASSERT() more appropriate given pool_prop_default should always have a label property?

Reply:
Fixed, thanks.